spec/components/custom/budgets/investments/form_component_spec.rb

```diff
@@ -0,0 +1,23 @@
+require "rails_helper"
+
+describe Budgets::Investments::FormComponent, type: :component do
+  include Rails.application.routes.url_helpers
+
+  let(:budget) { create(:budget) }
+
+  before do
+    allow(controller).to receive(:current_user).and_return(create(:user))
+    allow(request).to receive(:path_parameters).and_return(budget_id: budget.id)
+  end
+
+  it "hint fields are shown" do
```

Comment:
IMHO, where possible, the lines starting with `it` should make sense in English :thinking:. Reading "It hint fields are shown" is a bit strange.

```suggestion
  it "shows hint fields" do
```